ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/GenesisFileModule.java

```diff
@@ -101,20 +97,11 @@ static GenesisFileModule createGenesisModule(final File genesisFile) throws IOEx
   }
 
   private static GenesisFileModule createGenesisModule(final String genesisConfig) {
-    // duplicating work from JsonGenesisConfigOptions, but in a refactoring this goes away.
     final JsonObject genesis = new JsonObject(genesisConfig);
     final JsonObject config = genesis.getJsonObject("config");
-    if (config.containsKey("ethash")) {
-      return new MainnetGenesisFileModule(genesisConfig);
-    } else if (config.containsKey("ibft")) {
-      return new IBFTGenesisFileModule(genesisConfig);
-    } else if (config.containsKey("clique")) {
-      return new CliqueGenesisFileModule(genesisConfig);
-    } else if (config.containsKey("qbft")) {
-      return new QBFTGenesisFileModule(genesisConfig);
-    } else {
-      // default is mainnet
-      return new MainnetGenesisFileModule(genesisConfig);
+    if (config.containsKey("ibft") || config.containsKey("clique") || config.containsKey("qbft")) {
+      throw new RuntimeException("Only Ethash and Merge configs accepted as genesis files");
```

Comment:
is this related to this PR?

Reply:
This unused feature (the ability to calculate the fork from a genesis file) has a long cascading requirement that ultimately brings in all the RocksDB classes, and requires significantly more reflection config.  Nobody uses the feature because it is fragile and buggy.

Reply:
In short, these need to be removed to use GraalVM for evmtool.  I assert that no one will miss them.